independent-publisher-connectors/OpenAI GPT-4/apiDefinition.swagger.json

```diff
@@ -0,0 +1,695 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "OpenAI (V2)",

```

Comment:
@troystaylor can you please change the title, our backend is not accepting this title as is. can we please make it "OpenAI GPT4" or something similar?

Reply:
Would OpenIA V2 work?